upload_certificate.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import json
+import sys
+import base64
+from typing import Mapping, Sequence, Any
+
+import requests
+
+
+# build headers
+def build_cloudflare_request_headers(auth_email: str, auth_key: str) -> Mapping[str, str]:
+    return {
+        "Content-Type": "application/json",
+        "X-Auth-Key": auth_key,
+        "X-Auth-Email": auth_email
+    }
+
+
+# upload certificate to cloudflare
+def upload_certificate(zone_id: str, auth_email: str, auth_key: str, key: str, crt: str):
+    url: str = f"{CF_BASE_URL}/zones/{zone_id}"
```

Comment:
`CF_BASE_URL` should also be imported from a shared Python file.

Reply:
Done